geode-core/src/main/java/org/apache/geode/pdx/internal/json/PdxToJSON.java

```diff
@@ -85,7 +84,6 @@ public String getJSON() {
   private void enableDisableJSONGeneratorFeature(JsonGenerator jg) {
     jg.enable(Feature.ESCAPE_NON_ASCII);
     jg.disable(Feature.AUTO_CLOSE_TARGET);
-    jg.setPrettyPrinter(new DefaultPrettyPrinter());
```

Comment:
Is this the reason the json response we get for our management v2 rest call is pretty-printed?

Reply:
AFAIK this is only used for PDX queries.